content/css/concepts/colors/terms/rgb/rgb.md

```diff
@@ -55,3 +59,5 @@ h1 {
   color: rgb(230, 230, 250);
```

Comment:
This color is very light and is difficult to see. I would recommend changing the color and the image to something like rgb(124, 124, 242)